### PR TITLE
Feedback july 23rd engelbert

### PR DESCRIPTION
**Pull request by eldonmetz**
Closes # 51

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed
- [ ] I have selected a committee co-chair to review the PR

### CODE REVIEW

File: CPs/cp_nn6.adoc [hunk 1]
```diff
@@ -835,7 +835,7 @@ Table 3.52.2-1: Actor Roles
...
 
 |Device-Patient Association Consumer
-|The receiver of the verified and final association report.  The consumer initially receives current association status followed by updates in real-time on a connection established by the Manager. It may optionally send a filter dynamically in the form a HL7 query or that filter may be pre-configured. When a association report is successfully received, a commit-level accept acknowledgement must be returned the the Manager.
+|The receiver of the verified and final association report. The Consumer may optionally initiate a subscription by sending a message with filtering criteria, if any, to the Manager in the form a HL7 query. The subscription and filter may also be pre-configured on the Manager. The Consumer initially receives current association status followed by updates in real-time on a connection established by the Manager.  When a association report is successfully received, a commit-level accept acknowledgement must be returned to the Manager.
```
Comment (bilhar97) [inline CPs/cp_nn6.adoc:839]: "a association report" -> "an association report.."

File: CPs/cp_nn6.adoc [hunk 2]
```diff
@@ -919,9 +919,8 @@ This transaction is used by a Device Patient Association Consumer to access filt
...
-If the DEV-19 transaction is not supported, and the network connection between the Device-Patient Association Manager and Device-Patient Association Consumer is lost, 
-the Device-Patient Association Manager shall send DEV-52 messages for all current Device-Patient associations to the Device-Patient Association Consumer when network connectivity is restored. 
+
+If this message is not supported, MSA-1 shall contain the value CR, ERR-3 (HL7 Error Code) shall contain the value 200 (Unsupported Message Type), and ERR-4 (Severity) shall contain the value E. If the transaction is not supported, and the network connection between the Device-Patient Association Manager and Device-Patient Association Consumer is lost, the Device-Patient Association Manager shall send DEV-52 messages for all current Device-Patient associations to the Device-Patient Association Consumer when network connectivity is restored. This ensures the Device-Patient Consumer hasn’t missed any association events prior to the on-going feed of associations in real-time.
```
Comment (bilhar97) [inline CPs/cp_nn6.adoc:924]: "Device-Patient Consumer" ->"Device-Patient Association Consumer"

File: CPs/cp_nn6.adoc [hunk 3]
```diff
@@ -1045,7 +1044,7 @@ The management of the query and response connection between the Device-Patient A
 [.text-left]
 [underline]#Proposed Text:#
 [.text-left]
-The management of the query and response connection between the Device-Patient Association Consumer and the Device-Patient Association Manager in the case of an ongoing subscription is an implementation detail, but one practical method is for the Device-Patient Association Manager to maintain an open TCP listen port to accepts connections from one or more Device- Patient Association Consumer clients and then to open an individual TCP connection with each requester that persists as long as the client is connected and the query is valid (within its time limits, if any).
+The Device-Patient Association Manager is responsible for collecting, formatting and sending the requested information back to the querying Device-Patient Association Consumer according to the filtering specified in the query.
```
Comment (bilhar97) [inline CPs/cp_nn6.adoc:1047]: Suggestion: "..back to the querying Device-Patient..." -> "back to the Device-Patient.."

File: CPs/cp_nn6.adoc (hunk 1 above)
Comment (eldonmetz) [reply CPs/cp_nn6.adoc:839]: Made this change.

File: CPs/cp_nn6.adoc (hunk 3 above)
Comment (eldonmetz) [reply CPs/cp_nn6.adoc]: Thanks, changes committed.

**Review by bilhar97 — APPROVED** [review]

**Pull request merged**